pkg/image/validation_test.go

```diff
@@ -25,11 +25,12 @@ var _ = Describe("Validate Test IsValidImageFile", func() {
 				imageFileName: "test.tar.xz",
 			},
 		}
-		It("returns false when the suffix are invalid type or empty", func() {
-			for _, t := range tests {
-				Expect(IsValidImageFile(t.imageFileName)).Should(BeTrue())
-			}
-		})
+		for _, t := range tests {
+			name := t.imageFileName
+			It("returns false when the suffix are invalid type or empty", func() {
+				Expect(IsSupporedFileType(name)).Should(BeTrue())
```

Comment:
I forgot to change the tests when thos method name changed, sorry.
Related, we have an issue to support `make test`, #53

Reply:
Also, how were you able to compile test without _vendor/github.com/onsi_?

Reply:
sorry for didn't change the `Makefile`, I'm using goland(https://www.jetbrains.com/go/) which is a pretty simple for run unit-test just need to click the button, but maybe I missed some vendor import...will take a look tomorrow.